common/common.cpp

```diff
@@ -1,4 +1,6 @@
 #include "common.h"
+// Change JSON_ASSERT from assert() to GGML_ASSERT:
+#define JSON_ASSERT GGML_ASSERT
```

Comment:
Maybe we only need to define this macro before `#include "common.h"`, so inside common.h we no longer need to re-define this.

I'm not 100% sure, so please try to see if it works.

Reply:
In fact, I'm more concerned about introducing this in `common.h` since it is used by other examples (not just `server.cpp`). But after a second though, I think it's OK to keep it here. It doesn't seem to have much that impact on other examples.